cpp/core/shuffle/Spill.cc

```diff
@@ -73,7 +73,7 @@ void Spill::insertPayload(
 
 void Spill::openSpillFile() {
   if (!is_) {
-    GLUTEN_ASSIGN_OR_THROW(is_, arrow::io::MemoryMappedFile::Open(spillFile_, arrow::io::FileMode::READ));
+    GLUTEN_ASSIGN_OR_THROW(is_, arrow::io::ReadableFile::Open(spillFile_));
```

Comment:
Is the API implemented with buffered read? 

Not sure whether https://github.com/apache/arrow/blob/main/cpp/src/arrow/io/buffered.h may help here.

Reply:
Spill merge needn't buffer